src/Extractor/Validator.php

```diff
@@ -44,7 +45,20 @@ public function validateProperties(array $configProperties): Generator
 
     public function validateProfiles(array $configProfiles): Generator
     {
-        $allowProfiles = $this->gaApi->getAccountProfiles();
+        try {
+            $allowProfiles = $this->gaApi->getAccountProfiles();
+        } catch (ClientException $e) {
```

Comment:
tady odchytávám vyjímku pokud nejde stáhnout seznam profilů k ověření
seznam se stahuje z jiného API (z toho co se stahují properties) a to nemusí být u všech  custom oauth konfiguraci dostupné

Reply:
příklad - https://keboola.zendesk.com/agent/tickets/21058